api/src/handlers.js

```diff
@@ -25,11 +25,10 @@ const itemsHandler = (request, reply) => {
     )}'`;
     reply.code(statusCode).send(body);
   } else {
-    const dgraphClient = new DgraphClient(
-      new DgraphClientStub('http://localhost:8080')
-    );
+    const dgraphClient = new DgraphClient(new DgraphClientStub('http://localhost:8080'));
     const txn = dgraphClient.newTxn();
-    txn.query(mapRequest(request)).then(response => {
+    const { query, vars } = mapRequest(request);
```

Comment:
Like the swap to querying with vars, unfortunately it looks like Dgraph is currently not happy with the queries with vars, I think the keys here need to be prepended with a '$' symbol? 🤑

Reply:
Ah yup, good catch!

Reply:
af7ed303b302ce28dd5be40216d92d0ffc653a64.